src/i_video.c

```diff
@@ -1664,6 +1664,12 @@ void I_GetScreenDimensions (void)
 	WIDESCREENDELTA = ((SCREENWIDTH - NONWIDEWIDTH) >> crispy->hires) / 2;
 }
 
+// [crispy] calls native SDL vsync toggle
+void I_ToggleVsync (void)
+{
+    SDL_RenderSetVSync(renderer, crispy->vsync);
```

Comment:
This needs a build-time check for either `SDL >= 2.0.18` or a symbol availability check for `SDL_RenderSetVSync()` (also build-time). And then e.g.:

```c
#if SDL_VERSION_ATLEAST(2, 0, 18)
  SDL_RenderSetVSync(renderer, crispy->vsync);
#else
  I_ReInitGraphics(REINIT_RENDERER | REINIT_TEXTURES | REINIT_ASPECTRATIO);
#endif
```